### PR TITLE
Workout find intent added footer

### PR DESCRIPTION
**Pull request by mihirapatel**
Footer has next and previous buttons to show new or previous video pages and video page numbers

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/utils/WorkoutUtils.java [hunk 1]
```diff
@@ -41,11 +46,20 @@ private static JSONObject readJsonFromUrl(String url) throws IOException, JSONEx
...
   }
 
+  /**
+   * Sets YouTube Data API search by keyword parameters, creates URL, and passes URL into
+   * readJsonFromURL
+   *
+   * @param workoutLength for workout video length
+   * @param workoutType for workout video type
+   * @param youtubeChannel for workout channel
+   * @param numVideos for number of videos to get from search
```
Comment (marcusschwartz) [inline portfolio/src/main/java/com/google/sps/utils/WorkoutUtils.java:56]: docstring doesn't match the params list

File: portfolio/src/main/webapp/workout.js [hunk 2]
```diff
@@ -1,46 +1,77 @@
+indexStart = 0;
+indexEnd = 5;
```
Comment (abbymapes) [inline portfolio/src/main/webapp/workout.js:2]: If these are global, may want to use var identifier

File: portfolio/src/main/webapp/workout.js [hunk 3]
```diff
@@ -1,46 +1,77 @@
...
 
-function createVideoDivs(videos) {
+/**
+* Creates video page divs and adds extra video pages if possible
+*
+* @param videos JSON object of all videos
+* @param indexStart index of first video on page
+* @param indexEnd index of last video on page + 1
+*/
+function createVideoDivs(videos, indexStart, indexEnd) {
```
Comment (abbymapes) [inline portfolio/src/main/webapp/workout.js:18]: Can this function be broken up at all to smaller functions (if you are planning to do similar things with different displays)? Like createChannelTitle(), createButtons(), etc?

File: portfolio/src/main/java/com/google/sps/utils/WorkoutUtils.java (hunk 1 above)
Comment (mihirapatel) [reply portfolio/src/main/java/com/google/sps/utils/WorkoutUtils.java:56]: fixed in next commit

File: portfolio/src/main/webapp/workout.js (hunk 3 above)
Comment (mihirapatel) [reply portfolio/src/main/webapp/workout.js:18]: not planning on resusing for now, so going to leave as is but will creak it up if i do end up resusing!

**Pull request merged**